src/components/message-box/MessageBox.spec.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import MessageBox, {COLOR} from './MessageBox';
+import Box from '../box/Box';
+import {mount, shallow} from 'enzyme';
+
+describe('<MessageBox />', () => {
+  it('render', () => {
+    const messageBox = shallow(
+      <MessageBox>some text</MessageBox>
+    );
+
+    expect(messageBox.hasClass('sg-message-box')).toEqual(true);
+  });
+
+  it('error when no children', () => {
+    const spy = jest.spyOn(console, 'error');
+
+    console.error = jest.fn();
+    shallow(
+      <MessageBox />
+    );
+    expect(console.error.mock.calls).toHaveLength(1);
+
+    spy.mockRestore();
+  });
+
+  it('render <Box />', () => {
+    const messageBox = shallow(
+      <MessageBox>some text</MessageBox>
+    );
+    const box = messageBox.find(Box);
+
+    expect(box).toHaveLength(1);
+  });
+
+  it('sets <Box /> component color - blue-secondary', () => {
+    const messageBox = shallow(
+      <MessageBox color={COLOR.BLUE_SECONDARY} />
+    );
+
+    expect(messageBox.find(Box).props().color).toEqual('blue-secondary');
+  });
+
+  it('check if close button exists', () => {
+    const mockCallback = jest.fn();
+    const messageBox = mount(<MessageBox onClose={mockCallback} />);
+
+    expect(messageBox.find('.sg-message-box__close')).toHaveLength(1);
+  });
+
+  it('check when no close button', () => {
+    const messageBox = mount(<MessageBox />);
+
+    expect(messageBox.find('.sg-message-box__close')).toHaveLength(0);
+  });
+
+  it('check if padding-right is bigger when close button exists', () => {
+    const mockCallback = jest.fn();
+    const messageBox = mount(<MessageBox onClose={mockCallback} />);
+
+    expect(messageBox.find(Box).hasClass('sg-message-box--closable')).toEqual(true);
```

Comment:
checking like this is not only unit test, coz u test 2 components.

You could use shallow render and check props instead of hasClass